src/client/containers/Layout/index.tsx

```diff
@@ -92,9 +99,15 @@ export const Layout: FC = ({ children }) => {
   const collapsedSidebar = useAppSelector(SettingsSelectors.selectSidebarCollapsed);
   const previousAddress = usePrevious(selectedAddress);
   const previousNetwork = usePrevious(currentNetwork);
+  const selectedVault = useAppSelector(VaultsSelectors.selectSelectedVault);
   // const path = useAppSelector(({ route }) => route.path);
   const path = location.pathname.toLowerCase().split('/')[1] as Route;
 
+  let vaultName;
+  if (path === 'vault') {
+    vaultName = selectedVault?.displayName;
+  }
+
   // TODO This is only assetAddress on the vault page
```

Comment:
Should we do the same for Labs?

Reply:
Added note for then we merge the new view in 37090db0d7efc2706a2f90189ac28479f41cded6